mypy/stubgenc.py

```diff
@@ -217,7 +252,12 @@ def generate_c_type_stub(module: ModuleType,
         if not any(issubclass(b, base) for b in bases):
             bases.append(base)
     if bases:
-        bases_str = '(%s)' % ', '.join(base.__name__ for base in bases)
+        bases_str = '(%s)' % ', '.join(
+            strip_or_import(
+                '%s.%s' % (base.__module__, base.__name__),
+                module,
+                imports
+            ) for base in bases)
```

Comment:
I'd move this `)` to the previous line -- somehow for function calls (unlike list/dict displays) we like the close parenthesis right after the last argument. But the `for` clause should stay on its own line.